torch/_dynamo/variables/functions.py

```diff
@@ -229,12 +229,17 @@ def bind_args(self, parent, args, kwargs):
                     if cell in side_effects:
                         out = side_effects[cell]
                     else:
-                        closure_cell = GetItemSource(
-                            AttrSource(self.source, "__closure__"), idx
-                        )
-                        closure_cell_contents = AttrSource(
-                            closure_cell, "cell_contents"
-                        )
+                        if "trampoline_autograd" in self.fn.__name__:
```

Comment:
this is not very robust, right? It looks like we are in UserFunctionVariable -- a user could write a function named `trampoline_autograd_blah` and it could trigger this

Reply:
Yes, I think the way to do this will be direct fn compare once I pull it out into its own toplevel file, instead of being in this nested garbage location that also makes skipfiles gross.